hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1396,27 +1395,25 @@ protected void preWrite(String instantTime) {
    * records and hence is more suited to bulkInsert for write performance.
    *
    * @param instantTime    - Action instant time for this commit
-   * @param partitionType  - The MDT partition to which records are to be committed
+   * @param partitionName  - The MDT partition to which records are to be committed
    * @param records        - records to be bulk inserted
    * @param fileGroupCount - The maximum number of file groups to which the records will be written.
    */
   protected abstract void bulkCommit(
-      String instantTime, MetadataPartitionType partitionType, HoodieData<HoodieRecord> records,
+      String instantTime, String partitionName, HoodieData<HoodieRecord> records,
```

Comment:
Could you add more details to the docs of `partitionName`?  Is `partitionName` going to be different for secondary/functional index, e.g., each secondary/functional index definition has its own partition path?

Reply:
Synced offline that the `partitionName` is going to be different per index definition within secondary and functional index.